src/arch/xtensa/boot_loader.c

```diff
@@ -263,6 +291,7 @@ void boot_master_core(void)
 		platform_panic(SOF_IPC_PANIC_MEM);
 		return;
 	}
+	shim_write(SHIM_LDOCTL, SHIM_LDOCTL_LPSRAM_LDO_ON);
```

Comment:
Is it only needed for APL or for all cAVS platforms?

Reply:
On all we wish to enable LPSRAM. However our approach is - enable on demand therefore the #if defined(...)